CHANGELOG.md

```diff
@@ -0,0 +1,26 @@
+# Changelog
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html)
+
+## [Unreleased]
+### Added
+- Contrast Scan support
+
+### Removed
+- `profile` configuration which the Contrast server has not supported since before 3.7.7
+
+
+## [2.12] - 2021-03-09
+### Changed
+- Builds with JDK 8, 11, and 15
+- Targets JRE 7
```

Comment:
We aren’t using animalsniffer or testing on Java 7 at all right? I’m concerned we might break our Java 7 support pretty easily. Do we have a ticket to add animalsniffer or Java 7 to pipelines?

Reply:
We're dropping support for JRE 1.7. The Integrations Team already made this decision and have targeted 1.8 in the contrast-java-sdk library upon which this project depends. I'll make a note in the [Unreleased] changelog entry